cache/remotecache/export.go

```diff
@@ -21,6 +22,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	attrLayerCompression = "compression"
+	attrForceCompression = "force-compression"
+	attrCompressionLevel = "compression-level"
+)
+
```

Comment:
I think exporter-specific options shouldn't be defined here.

Reply:
Thanks for the reply, will fix it.

Reply:
Fixed it by dropping the commit first.